src/components/calcite-tile/calcite-tile.tsx

```diff
@@ -1,5 +1,10 @@
 import { Component, Fragment, h, Prop, VNode } from "@stencil/core";
+import { SLOTS } from "./resources";
 
+/**
+ * @slot content-start - A slot for adding non-actionable elements before the tile content.
```

Comment:
Should we doc default slot information?

Reply:
For the content, tile uses the `heading`/`description` props, so there's no default slot at the moment. Or were you suggesting a default slot that has the heading and description as its default content?

Reply:
Oh interesting. Is it odd if there isn't a default slot but there are named slots?

Reply:
I think its probably fine. Nvm then!